content/docs/programs/immunizations.md

```diff
@@ -94,13 +71,40 @@ To add a new Vaccine Course, click the `Add vaccine course` button in the top ri
 
 ![vaccine course: add button](/docs/programs/images/vaccine_course_add_button.png)
 
-This will open a new window, where you can enter the name of the new Vaccine Course.
+This will open a new window, where you can enter the details of the new Vaccine Course, e.g. Vaccine course name, wastage rate, dose schedule.
 
 ![vaccine course: create modal](/docs/programs/images/vaccine_course_add.png)
 
-- Click `OK` once you are happy with your changes
-  - You will be redirected to the detail view of your newly created Vaccine Course
-- OR: Click `Cancel` to close the window if you've changed your mind
+#### Doses
+
+To add doses to the Vaccine Course, click the `(+) Dose` button. This will add a new row to the `dose schedule` table, where the details of the dose can be entered, including when the dose can administered.
+
+![vaccine course: dose edit table](/docs/programs/images/vaccine_course_dose_edit.png)
+
+The Dose Schedule table is divided into 6 columns:
+
+| Column                  | Description                                                                                                              |
+| :---------------------- | :----------------------------------------------------------------------------------------------------------------------- |
+| **Dose #**              | The number of the dose in the schedule                                                                                   |
+| **Label**               | The label of the dose, to appear in the [Vaccination Card](/docs/programs/program-module#vaccination-cards)              |
+| **From age**            | Minimum recommended age to receive the dose (in years and months)                                                        |
+| **To age**              | Maximum recommended age to receive the dose (in years and months)                                                        |
+| **Custom age label**    | Custom age label to be displayed in the Vaccination Card. If not defined, the default is the `From age` (e.g. `6 years`) |
+| **Min interval (days)** | Minimum number of days before the patient should receive the next dose                                                   |
+
+<div class="note">
+  If custom age labels are used, ensure that label is the same across all doses in the Immunization Program with the same <code>From age</code>. For example, if you want to replace "0 months" with "Birth", all doses across all Vaccine Courses with a <code>From age</code> of 0 months will need the custom age label of "Birth". This ensures correct grouping of doses in the Vaccination Card.
+</div>
```

Comment:
Also added this note per chat with Mariya and Richard... let me know your thoughts 🤷‍♀️